src/Schema/Events/Click.ts

```diff
@@ -373,8 +373,8 @@ export interface ClickedBid {
  *  ```
  *  {
  *    action: "clickedChangePaymentMethod",
- *    context_module: "OrdersReview",
- *    context_page_owner_type: "orders-review",
+ *    context_module: "OrdersReview" | "OrdersHistory",
```

Comment:
Added this as an example of how we plan to use this on the orders history page. Think we'll put specific `order_id` from the row that displays update `update payment` button  into `context_page_owner_id` - which is theoretically not correct as `ordersHistory` page does not include id. But this id will point to the corresponding row. You think it's acceptable @daytavares ? Alternatively we can add `item_id` to this schema to be more proper with our assignment.

Reply:
I think this is good, so we can track the `order_id`.